extra/intl-extra/tests/Fixtures/format_date.test

```diff
@@ -15,11 +15,11 @@
 return [];
 --EXPECT--
 Aug 7, 2019, 11:39:12 PM
-7 août 2019 23:39:12
+7 août 2019 à 23:39:12
 23:39
 07/08/2019
-mercredi 7 août 2019 23:39:12 UTC
-11 oclock PM, GMT
+mercredi 7 août 2019 à 23:39:12 heure d’été d’Europe centrale
+11 oclock PM, Central European Summer Time
```

Comment:
IMHO this is just based on "newer" language packs. Maybe this is not failing on travis, because the use an older build of the distro or language pack 🤔